notebooks/06_Task_Arithmetics.ipynb

```diff
@@ -0,0 +1,535 @@
+{
```

Comment:
regarding the warnings produced by HF pipelines: There's a hack to suppress these here: https://github.com/huggingface/api-inference-community/blob/0b6a3a9494b934f07d6d8ffa344ba93d1daf8c4d/docker_images/adapter_transformers/app/pipelines/base.py#L28-L33

Reply:
Every way I found to suppress these logs was way too long for the notebook so I left it like it is & added a note for the readers that this error log can be ignored.